package.json

```diff
@@ -2,7 +2,7 @@
     "name": "berry-material-react-free",
     "version": "3.0.1",
     "private": true,
-    "homepage": "https://berrydashboard.io/free",
+    "homepage": "https://berrydashboard.io",
```

Comment:
Could you make an a JIRA issue for domain name and reminder for us to change this up in the future?